app/models/concerns/ss/addon/twitter_setting.rb

```diff
@@ -25,4 +25,8 @@ def twitter_card_options
   def twitter_card_enabled?
     twitter_card.present? && twitter_card != 'none'
   end
+
+  def twitter_token_enabled?
+    twitter_consumer_key.present? && twitter_consumer_secret.present? && twitter_access_token.present? && twitter_access_token_secret.present?
```

Comment:
__[rubocop]__
* Line is too long. [142/130] [:ref](https://github.com/bbatsov/ruby-style-guide#80-character-limits)

<!-- Comment by [SideCI](https://sideci.com) -->